### PR TITLE
Run systemd-hwdb when building an image

### PR DESCRIPTION
**Pull request by DaanDeMeyer**

### CODE REVIEW

File: mkosi/__init__.py [hunk 1]
```diff
@@ -1478,6 +1478,11 @@ def run_preset(state: MkosiState) -> None:
         run(["systemctl", "--root", state.root, "preset-all"])
 
 
+def run_hwdb(state: MkosiState) -> None:
+    with complete_step("Generating hardware database"):
+        run(["systemd-hwdb", "--root", state.root, "--usr", "--strict", "update"])
```
Comment (behrmann) [inline mkosi/__init__.py:1483]: Is any `systemd-hwdb` fine or should we advise to build a recent one like we do for other tools?

**Review by behrmann — COMMENTED** [review]
LGTM, one question

File: mkosi/__init__.py (hunk 1 above)
Comment (keszybz) [reply mkosi/__init__.py:1483]: Any should be fine. I don't think it changed in living memory, except for support for `query --root`. But `update --root` worked always.

Comment (keszybz) [reply mkosi/__init__.py:1483]: `--strict` is from 2018, and `--usr` since the first version.

**Pull request merged**